README.md

```diff
@@ -6,23 +6,23 @@ This module provides a easy way to apply cursor-based pagination to your Ecto Qu
 
 ## Usage:
 
-  1. Add macro to your repo
+1. Add macro to your repo
 
-     ```elixir
-     defmodule MyRepo do
-        use Ecto.Repo, otp_app: :my_app
-        use Ecto.Paging.Repo # This string adds `paginate/2` and `page/3` methods.
-      end
-     ```
+```elixir
```

Comment:
Why did you remove padding here? For me, it looked better when code was inside section, not outside of it.

Reply:
No problem, just saw the code formatting block wasn't taking on the Github readme. I don't have any strong opinions